rsrcdump/structtemplate.py

```diff
@@ -118,8 +123,8 @@ def pack(self, obj: Any) -> bytes:
                 buf += self.pack_record(item)
             return buf
 
-    def pack_record(self, json_obj: Any) -> bytes:
-        def process_json_field(_field_format, _field_value):
+    def pack_record(self, json_obj: list[str] | dict[str, str] | str) -> bytes:
```

Comment:
Annotating json blobs is tricky here as well. There's no guarantee the user's JSON will match `dict[str, str]`.

Reply:
Updated to be more accurate in e907f63